help/compute/compute.md

```diff
@@ -443,12 +443,33 @@ Automatic outliers detection Manual outliers markup and annotation Used as an in
 
 ### Sensitivity analysis
 
-* Sample inputs:
-  * by specified number of samples
-  * by a specified distribution or within a range
-  * for a specified set of scalar inputs and/or columns of the matrix input
-* Produce variability analysis for outputs based on the sampled inputs
-* Visualize the results of analysis with Datagrok viewers
+Study what happens to a function when varying its inputs. Datagrok provides investigation of how uncertainty in its output can be divided to different sources of uncertainty in the inputs.
+
+To run analysis, click on the icon "Run sensitivity analysis", choose a method, specify inputs & outputs, and press "RUN".
+
+Available types of analysis:
+
+* Grid
```

Comment:
I would simplify and put the descriptions inline. Something like that:

* **Grid**: varying parameters in a grid with the specified step.
* **Sobol**: (link): decomposes output variance into fractions, which can be attributed to inputs
...

Reply:
Note the use of the bolder font; attract attention to the most important pieces

Reply:
Done